src/naf-dialog-adapter.js

```diff
@@ -596,12 +782,13 @@ export default class DialogAdapter {
     // Close protoo Peer, though may already be closed if this is happening due to websocket breakdown
     if (this._protoo && this._protoo.connected) {
       this._protoo.close();
+      this.scene.emit("rtc_event", { level: "info", tag: "Signaling", msg: "[close]" });
     }
 
     // Close mediasoup Transports.
-    if (this._sendTransport) this._sendTransport.close();
+    if (this._sendTransport && !this._sendTransport._closed) this._sendTransport.close();
```

Comment:
Was this causing some sort of error?

Reply:
I think I got some exception at some point but I can't reproduce it anymore so I'm removing it.